### PR TITLE
UPDATE: Updated Toulligqc to v2.5.6 + changed input section

### PR DESCRIPTION
**Pull request by Salome-Brunon**
Updated ToulligQC to v2.5.6 to fix some graph issues in the report. 
Reformatted the input options to facilitate the module integration in pipelines and subworkflows.

### CODE REVIEW

**Comment by Salome-Brunon** [comment]
@jfy133 Hello! I have made some changes to my module to make it easier to implement it in nf-core pipelines. 
As you know my module quite well from the multiple pull requests 😅 , I thought I would add you as reviewer, hope you don't mind!

File: modules/nf-core/toulligqc/main.nf [hunk 1]
```diff
@@ -4,18 +4,19 @@ process TOULLIGQC {
...
-    path "*/images/plotly.min.js"       , emit: plotly_js
+    tuple val(meta), path(ontfile)
+
 
+    output:
+    tuple val(meta), path("*/*.data")                   , emit: report_data
+    tuple val(meta), path("*/*.html")                   , emit: report_html, optional: true
+    tuple val(meta), path("*/images/*.html")            , emit: plots_html
+    tuple val(meta), path("*/images/plotly.min.js")     , emit: plotly_js
     path "versions.yml" , emit: versions
```
Comment (SPPearce) [inline modules/nf-core/toulligqc/main.nf:20]: Can you align this , emit with the others please.

File: modules/nf-core/toulligqc/main.nf [hunk 2]
```diff
@@ -25,15 +26,13 @@ process TOULLIGQC {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
 
-    def seq_summary_input = seq_summary ? "--sequencing-summary-source ${seq_summary}" : ""
-    def fastq_input = fastq ? "--fastq ${fastq}" : ""
-    def bam_input = bam ? "--bam ${bam}" : ""
+    def input_file = ("$ontfile".endsWith(".fastq") || "$ontfile".endsWith(".fastq.gz") || "$ontfile".endsWith(".fq") || "$ontfile".endsWith(".fq.gz")) ? "--fastq ${ontfile}" :
+        ("$ontfile".endsWith(".txt") || "$ontfile".endsWith(".txt.gz")) ? "--sequencing-summary-source ${ontfile}" :
+        ("$ontfile".endsWith(".bam")) ? "--bam ${ontfile}" : ''
```
Comment (SPPearce) [inline modules/nf-core/toulligqc/main.nf:31]: Would this work with cram input?
Should this fail at the end?

**Review by SPPearce — COMMENTED** [review]
A few comments, but generally looks good.

File: modules/nf-core/toulligqc/main.nf (hunk 2 above)
Comment (Salome-Brunon) [reply modules/nf-core/toulligqc/main.nf:31]: No, ToulligQC doesn't deal with CRAM files.
If no files/ incorrect files are given, the module will fail yes!

**Review by SPPearce — APPROVED** [review]

**Pull request merged**